### ISSUE TITLE
Binding a record to an inline map with 'key' and 'target' attributes is not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#61]
```
Given the following mapping file:

  <stream name="stream" format="csv">
    <group name="group" class="map">
      <record name="record" collection="map" key="name" target="value">
        <field name="name" />
        <field name="value" />
      </record>
    </group>
  </stream>

I would expect a map where:

"record" -> { "name1" -> "value1", "name2" -> "value2" }
```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 16 Feb 2013 at 4:55

**Comment by GoogleCodeExporter** [comment]
```
Reviewing the code indicates this was intentionally not supported for now.  
Changing to enhancement.
```

Original comment by `kevin.s...@gmail.com` on 16 Feb 2013 at 5:27
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Note the expected input for the example above is:

name1,value1
name2,value2
```

Original comment by `kevin.s...@gmail.com` on 16 Feb 2013 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.0.4.  Snapshot JAR attached.
```

Original comment by `kevin.s...@gmail.com` on 16 Feb 2013 at 9:22
- Changed state: **Started**
- Added labels: **Milestone-2.0.4**

Attachments:
- [beanio.jar](https://storage.googleapis.com/google-code-attachments/beanio/issue-61/comment-3/beanio.jar)

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 6 Mar 2013 at 3:14
- Changed state: **Fixed**
